### PR TITLE
Contrast issue in Continue Studying text on revision screen #4741

### LINKED ISSUE #4741: Contrast issue in Continue Studying text on revision screen
**Describe the bug**

The "Continue Studying"  text on the revision screen contrast issue in dark mode when run through the accessibility scanner.

**To Reproduce**
1. Go to Home screen and select any topic
2. Go to revision tab
3. Start any revision card in dark mode.
4. Scroll to the bottom. Continue Studying text has very low visibility.

**Environment**
- Device/emulator being used: <Vivo V25 pro>
- Android or SDK version (e.g. Android 5 or SDK 21): Android 12
- App version (you can get this through system app settings or via the admin controls menu in-app): 0.10-beta-5e64fae55e

### PR DESCRIPTION
**Pull request by iampradeep-hr**
## Explanation
##Fixes #4741 

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #4741 : " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only
<table role="table">
<thead>
<tr>
<th></th>
<th>Without changes</th>
<th>With changes</th>
</tr>
</thead>
<tbody>
<tr>
<td>Revision Card</td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/81664507/204736752-02ee1367-7195-4668-9988-7e0a36f086d8.jpeg">image</a></td>
<td><a target="_blank" rel="noopener noreferrer nofollow" href="https://user-images.githubusercontent.com/81664507/204736312-5ae16728-38bb-4dfa-aeda-c3258cac2981.jpg">image</a></td>
</tr>
</tbody>

### CODE REVIEW

File: app/src/main/res/layout/topic_lessons_fragment.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/oppia_background">
+    android:background="@color/component_color_topic_revision_fragment_background_color">
```
Comment (rt4914) [inline app/src/main/res/layout/topic_lessons_fragment.xml]: @iampradeep-hr Why is this change in this PR? This PR is focused on revision card only, right? If this change is needed then do add screenshot of topic-lessons too.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@iampradeep-hr PTAL. Thanks @MohitGupta121

File: app/src/main/res/layout/topic_lessons_fragment.xml (hunk 1 above)
Comment (iampradeep-hr) [reply app/src/main/res/layout/topic_lessons_fragment.xml]: It was not having dark mode, seems someone is already working on it. so i will drop it and revert back.

**Comment by iampradeep-hr** [comment]
PTAL

**Review by iampradeep-hr — COMMENTED** [review]
Reverted back android:backround="@color/oppia_background"

**Comment by MohitGupta121** [comment]
@iampradeep-hr you no need to mark as resolved review changes. This was done by the reviewer.

**Comment by iampradeep-hr** [comment]
> @iampradeep-hr you no need to mark as resolved review changes. This was done by the reviewer.

Oh sorry, I am new to it. I thought I should have to do it. I will remember it from next time.

File: app/src/main/res/values/component_colors.xml [hunk 2]
```diff
@@ -289,6 +290,8 @@
   <color name="component_color_audio_fragment_seekbar_color">@color/color_palette_seekbar_progress_background_color</color>
   <!-- Topic Revision Fragment -->
   <color name="component_color_topic_revision_fragment_background_color">@color/color_palette_topic_revision_background_color</color>
+   <!-- Topic Lesson Fragment -->
```
Comment (MohitGupta121) [inline app/src/main/res/values/component_colors.xml]: @iampradeep-hr if this is not part of your issue, so remove this color also.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by iampradeep-hr** [comment]
@MohitGupta121 OK I removed it.

**Comment by MohitGupta121** [comment]
Okay Thanks @iampradeep-hr

**Review by rt4914 — CHANGES_REQUESTED** [review]
Please apply similar change in the landscape file of `revision_card_fragment.xml`
Screenshot 2022-11-30 at 10 58 10 PM

**Comment by iampradeep-hr** [comment]
Done added it in the sw600dp layout as well

File: app/src/main/res/values/component_colors.xml (hunk 2 above)
Comment (iampradeep-hr) [reply app/src/main/res/values/component_colors.xml]: removed it

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks

**Pull request closed without merging**